docs/troubleshooting/troubleshoot_deployment.md

```diff
@@ -8,6 +8,18 @@ hide_title: true
 
 This document provides 'notes' on frequently encountered deployment issues. The issues reported are organized by relevant subsection.
 
+## Terraform Init
```

Comment:
This isn't really specific to `terraform init`. I would make the header `Add logging output` or something